hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -947,45 +952,12 @@ private TcpClientConnection onAuthenticated(TcpClientConnection connection,
             connection.setRemoteUuid(response.memberUuid);
             connection.setClusterUuid(response.clusterId);
 
-            if (tpcPorts == null || tpcPorts.isEmpty()) {
+            if (!isAltoAwareClient || tpcPorts == null || tpcPorts.isEmpty()) {
                 logger.info("TPC Client: disabled, no TPC ports detected");
```

Comment:
```suggestion
                logger.debug("TPC Client: disabled, no TPC ports detected");
```
For a client which did not enable the TPC, why would you write at info level? It sounds not relevant and that log is enabled by default.

Reply:
These are not introduced by this PR but let me fix it